lib/handlers/error-pages.js

```diff
@@ -9,12 +9,36 @@ function handler (err, req, res, next) {
 
   var ldp = req.app.locals.ldp
 
-  if (err.status === 401 &&
-      req.accepts('text/html') &&
-      ldp.auth === 'oidc') {
-    debug('401 error - redirect to Select Provider')
-    res.status(err.status)
-    redirectToLogin(req, res, next)
+  if (ldp.auth === 'oidc') {
+    let statusCode = err.status || err.statusCode || 500
```

Comment:
I remember. The reason I added it here is that this is the last place to catch it (if some other module forgot to specify a status code for their error), otherwise it gets a '0 is an invalid status code' error. I can take it out tho.